src/FSharp.Data.GraphQL.Server.AspNetCore/Giraffe/HttpHandlers.fs

```diff
@@ -49,19 +38,17 @@ module HttpHandlers =
         setHttpHeader "Access-Control-Allow-Origin" "*"
         >=> setHttpHeader "Access-Control-Allow-Headers" "content-type"
```

Comment:
@xperiandri it's not up to this code to set these headers here.

Reply:
I agree, it must be moved